src/suresh/SwappingNumbersUsingOtherVariable.java

```diff
@@ -0,0 +1,53 @@
+package suresh;
+
+public class SwappingNumbersUsingOtherVariable {
+
+		void swapNumbersWithTemp(int x,int y) {
+			int temp=x;
+			x=y;
+			y=temp;
+			System.out.println("========Swapping Numbers using Temp=========");
+			System.out.println("X Value is: "+x);
+			System.out.println("Y Value is: "+y);
+			
+		}
+		void swapStringWithTemp() {
+			String firstname ="Suresh";
+			String lastname ="Kumar";
+			String Name = firstname;
+			firstname = lastname;
+			lastname = Name;
+			System.out.println("========Swapping String using Temp=========");
+			System.out.println("firstname  is: "+firstname);
+			System.out.println("lastname  is: "+lastname);
+		}
+		void swapStringWithOutUsingTemp() {
+			String firstname ="Suresh";
+			String lastname ="Kumar";
+			String substr = "";
+			substr = firstname.substring(0, 6);
```

Comment:
Hard coding not allow
Pls refer yogita's code

https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/yogita/SwapString.java